app/models/manageiq/providers/amazon/storage_manager/ebs/refresh_parser_inventory_object.rb

```diff
@@ -41,6 +41,10 @@ def parse_volume(volume)
       :base_snapshot         => inventory_collections[:cloud_volume_snapshots].lazy_find(volume['snapshot_id']),
       :availability_zone     => inventory_collections[:availability_zones].lazy_find(volume['availability_zone'])
     }
+
+    link_volume_to_disk(volume_hash, volume['attachments'])
```

Comment:
Because storage manager is not aware of the ephemeral disks, I kept it in the cloud manager (basically, ephemeral disks are only created because the flavor itself specifies how many local disks there are).